### DISCUSSION TITLE
running jobs in the background?

### DISCUSSION THREAD

**Discussion by anarcat** [koreader/koreader#8003] [Q&A] [answered]
Is there a way to start jobs in the background? I found the Terminal stuff, but commands run in the foreground and basically hang koreader.

I'm trying to find an elegant way to start syncthing from koreader, as a background job.

I found the [background runner plugin](https://github.com/koreader/koreader/wiki/Background-runner) but its documentation is... rather minimal. :)

**Comment by poire-z** [comment, marked as answer]
You can run shell scripts from the File browser since #5804 - I'm sure that inside a shell script, you can use tricks to double fork/exec and detach the scripts from KOReader so it gets a life of its own.
(Forget about this dreadful plugin, and the terminal is so-so.)

**Reply by anarcat** [reply]
ah, that makes sense, i'll give that a shot and report back, thanks!

**Reply by anarcat** [reply]
I used this, and it works great, thanks. It would still be nice to have a way to see the output of a job, but i guess I should throw that stuff in a logfile or something. :)

Details in https://anarc.at/hardware/tablet/kobo-clara-hd/ and shouts out to the koreader folks, you do indeed rock. :)

**Reply by NiLuJe** [reply]
Depending on verbosity, you could technically send the output to the screen w/ fbink, c.f., how we handle zsync2
